### mission params ###
mission:
  # waypoint units in [[m], [m]]
  waypoints: [[2.0, 0.0], [3.0, 4.0], [3.0, 7.0],
              [7.0, 7.0], [7.0, 11.0]]
  distance_margin: 0.15   # [m]
### controller params ###
controller:
  # controller gains
  Kp: [0.76, 1.88]  # proportional gains [linear, angular]
  Kd: [0.07, 0.16]  # derivative gains [linear, angular]
